cogrun.py

```diff
@@ -15,15 +16,15 @@ def create_temporary_copy(src_path):
     shutil.copy2(src_path, temp_path)
     return temp_path
 
-class BasePixrayPredictor(cog.Predictor):
+class BasePixrayPredictor(BasePredictor):
     def setup(self):
         print("---> BasePixrayPredictor Setup")
         os.environ['TORCH_HOME'] = 'models/'
 
-    # Define the input types for a prediction
-    @cog.input("settings", type=str, help="Default settings to use")
-    @cog.input("prompts", type=str, help="Text Prompts")
-    def predict(self, settings, **kwargs):
+    def predict(self, 
+        settings: str = Input(description="Default settings to use"),
+        prompts: str = Input(description="Text prompts", default=None),
+    **kwargs) -> Iterator[str]:
```

Comment:
Should this be `Iterator[Path]`?

Reply:
Yep. Fixed in https://github.com/pixray/pixray/pull/50/commits/fe66a1c94b63fdea9202dac7e814f25376bfbeae